Automattic/bugomattic
# Add remaining Tracks events

#### What Does This PR Add/Change?

Adds the actual triggering of the remaining Tracks events on the client!

See this P2 for the full list of events: pciE2j-1LC-p2

** One unique note worth mentioning: Apart from page load, I've tried to tie every event to a direct user action (like actually clicking on the page). This means that loading a URL with existing state or using browser back/forward will _not_ trigger track events. I think this will get us a much more accurate picture of how users are using the app! Otherwise, we could get a lot of event spam in confusing orders.

#### Testing Instructions

You can see the full list of events here: pciE2j-1LC-p2

I have...
1. Gone through the app flow locally and verified all events in the console
2. Added unit tests for all events

You can verify in the console as well!

#### Issues



Related to #  
Closes #
